enigma/sw/rotor_test.cpp

```diff
@@ -0,0 +1,68 @@
+#include "rotor.cpp"
```

Comment:
Kaller du fila for rotor_tb.cpp. I testene jeg lagde så har jeg gått for et mønster der alle test cases er i sin egen funksjon med navne-prefiks test_, og kaller på disse i main-funksjonen i testbenken. Scanf vil vi ikke ha